benchmarks/generate.py

```diff
@@ -31,7 +31,7 @@ def create_file_of_given_size(path: str, size: int) -> None:
     """
     file_path = pathlib.Path(path)
     file_path.parent.mkdir(parents=True, exist_ok=True)
-    chunk_size = 8192
+    chunk_size = 1048576
```

Comment:
This change wasn't benchmarked (but also isn't required to have good performance), but changed for unity.